spec/requests/api/v1/unauthenticated_user_request_spec.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "Unauthenticated Access" do
```

Comment:
You mixed things from the 2 issues, more attention next time, please dont repeat this mistake